.github/workflows/pytest_benchmark_comment.yml

```diff
@@ -3,6 +3,9 @@ on:
   pull_request:
     branches:
       - master
+    paths:
+      - splink/**
+      - tests/**
```

Comment:
I think for this workflow we probably don't need to run it on changes to `tests/`, but maybe we could on changes to `benchmarking/` (which will be pretty rare)